### ISSUE TITLE
Min(symbol) doesnot work

### ISSUE THREAD

**Issue by jinqinn** [zeroSteiner/rule-engine#68]
Example:
```
items = [1, 2, 3, 4, 5, 6, 7, 8, 9]
rule = rule_engine.Rule("$min(items)")
```
failed with TypeError:
can not map python type 'SymbolExpression' to a compatible data type

**Comment by jinqinn** [comment]
it works after committing the below code in FunctionCallExpression.__init__:
self._validate_function(function_type, arguments)

it is a bug ?

**Comment by zeroSteiner** [comment]
This was a bug. Thanks for reporting it. I was able to reproduce the issue and get it fixed. You can expect the fix to be included in the next release, which I'll post in the next few days.

**Comment by zeroSteiner** [comment]
This issue was fixed in [version 4.0.1](https://github.com/zeroSteiner/rule-engine/releases/tag/v4.0.1) which is now available on [PyPi](https://pypi.org/project/rule-engine/4.0.1/). Thank you for bringing it to my attention.
